.github/workflows/ci-cd-pull-request.yml

```diff
@@ -19,10 +19,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Set up env
-        uses: the-guild-org/shared-config/setup@main
-        with:
-          nodeVersion: 20
-          packageManager: pnpm
```

Comment:
How does it know which version of Node to use if we don't specify it? (As for pnpm, I guess it either looks at which lockfile exists or at package.json's `packageManager`.)

Reply:
`shared-config/setup@v1` looks `packageManager` field to understand which package manager and version to use.
and about Node.js it looks `.node-version` or `.nvmrc` file. @saihaj @enisdenjo correct me if I am wrong

Reply:
> shared-config/setup@v1 looks packageManager field to understand which package manager and version to use

Yes.

>  Node.js it looks .node-version or .nvmrc file

Not automatically, you have to provide either the `node-version` or `node-version-file` to the action.